### PR TITLE
Brendan/algorithm dev

### PR DESCRIPTION
**Pull request by Brendanlai**
merging another version of the algorithm

### CODE REVIEW

**Comment by Brendanlai** [comment]
gonna need to resolve some conflicts obviously, after the final approval can someone just help me figure out to resolve it through the web editor or however else is easier?

File: backend/Controllers/algorithmController.js [hunk 1]
```diff
@@ -249,5 +486,6 @@ exports.createSpotifyPlaylist = async (req, res) => {
  * @param {*} res
  */
 exports.getGroupPlaylist = async (req, res) => {
-    console.log("EMPTY");
+  console.log("EMPTY");
+  res.json({ m: "lmao" });
```
Comment (KevinetPeng) [inline backend/Controllers/algorithmController.js:490]: lmao

**Review by KevinetPeng — APPROVED** [review]
Looks good to me. I trust the algo expert

**Review by Remyzh — APPROVED** [review]

**Comment by Brendanlai** [comment]
v1.2 is in :)

**Pull request merged**